src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import NavBar from './components/NavBar'
+import { SideBar } from './components/SideBar'
+import Body  from './components/Body'
+import Card from './components/Card'
+import  Footer  from './components/Footer'
+
+
+function App() {
+  return (
+    <>
+    <div className=' w-full h-screen'>
```

Comment:
Work on your indentation, you can install prettier to help you handle the formatting